src/components/NotFound.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { skinnable } from 'revenge';
+
+@skinnable()
+
```

Comment:
stylistic comment: this should go right on top of the class declaration (no empty line in between)